### ISSUE TITLE
Integration test fails even though chef appears to exit successfully

### ISSUE THREAD

**Issue by aramprice** [pivotal-sprout/sprout-vim#8]
From: https://travis-ci.org/pivotal-sprout/sprout-vim/
```
[ SNIP ]
 * cookbook_file[/Users/travis/.vimrc.local] action create_if_missing[2016-06-13T21:52:48+00:00] INFO: Processing cookbook_file[/Users/travis/.vimrc.local] action create_if_missing (sprout-vim::config line 51)
[2016-06-13T21:52:48+00:00] INFO: cookbook_file[/Users/travis/.vimrc.local] created file /Users/travis/.vimrc.local
    - create new file /Users/travis/.vimrc.local[2016-06-13T21:52:48+00:00] INFO: cookbook_file[/Users/travis/.vimrc.local] updated file contents /Users/travis/.vimrc.local
    - update content in file /Users/travis/.vimrc.local from none to ef33f2
    --- /Users/travis/.vimrc.local	2016-06-13 21:52:48.000000000 +0000
    +++ /Users/travis/.chef-.vimrc.local20160613-21225-1nz9gef	2016-06-13 21:52:48.000000000 +0000
    @@ -1 +1,8 @@
    +" This file gets loaded after all of the other vim configuration.
    +" You can put machine-specific overrides here.
    +
    +" set guifont=Source\ Code\ Pro:h18
    +
    +" set background=dark
    +" colorscheme solarized[2016-06-13T21:52:48+00:00] INFO: cookbook_file[/Users/travis/.vimrc.local] owner changed to 501
    - change owner from '' to 'travis'
[2016-06-13T21:52:48+00:00] INFO: Chef Run complete in 348.517984 seconds
[2016-06-13T21:52:48+00:00] INFO: Skipping removal of unused files from the cache
Running handlers:
[2016-06-13T21:52:48+00:00] INFO: Running report handlers
Running handlers complete
[2016-06-13T21:52:48+00:00] INFO: Report handlers complete
Chef Client finished, 11/17 resources updated in 06 minutes 05 seconds
F
Failures:
  1) sprout-vim installs MacVim, adds the .vim directory, and installs tmux
     Failure/Error: expect(system('soloist')).to be_true
       expected true to respond to `true?` or perhaps you meant `be true` or `be_truthy`
     # ./spec/integration/cookbook_spec.rb:15:in `block (2 levels) in <top (required)>'
Finished in 6 minutes 20 seconds (files took 0.15834 seconds to load)
1 example, 1 failure
Failed examples:
rspec ./spec/integration/cookbook_spec.rb:14 # sprout-vim installs MacVim, adds the .vim directory, and installs tmux
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby -I/Users/travis/build/pivotal-sprout/sprout-vim/tmp/ruby/2.0.0/gems/rspec-core-3.4.4/lib:/Users/travis/build/pivotal-sprout/sprout-vim/tmp/ruby/2.0.0/gems/rspec-support-3.4.1/lib /Users/travis/build/pivotal-sprout/sprout-vim/tmp/ruby/2.0.0/gems/rspec-core-3.4.4/exe/rspec --pattern spec/integration/\*\*/\*_spec.rb failed
The command "./sprout exec rake ci" exited with 1.
Done. Your build exited with 1.
```

**Comment by nertzy** [comment]
Fixed by #9

**Comment by aramprice** [comment]
awesome, thanks!
